Formula/brew-pip.rb

```diff
@@ -9,4 +9,8 @@ class BrewPip < Formula
   def install
     bin.install "bin/brew-pip"
   end
+
+  test do
+    system "#{bin}/brew-pip", "help"
```

Comment:
Can’t we write a test that does something more substantial than e.g. `--help` or `--version`?